src/WinUI/ArcGISRuntime.WinUI.Viewer/SamplePage.xaml.cs

```diff
@@ -82,5 +98,44 @@ private async void MarkdownText_LinkClicked(object sender, LinkClickedEventArgs
         {
             await Launcher.LaunchUriAsync(new Uri(e.Link));
         }
+
+        private void ScreenshotKeyDown_Event(object sender, KeyRoutedEventArgs e)
+        {
+            if (e.Key == Windows.System.VirtualKey.End)
+            {
+                GetJpgImage(SampleContainer);
+            }
+        }
+
+        private void GetJpgImage(UIElement source)
+        {
+            ContentControl sampleContainer = (ContentControl)source;
+            int actualHeight = (int)sampleContainer.ActualHeight;
+            int actualWidth = (int)sampleContainer.ActualWidth;
+
+            double scale = ScreenshotManager.ScreenshotSettings.ScaleFactor.HasValue ? ScreenshotManager.ScreenshotSettings.ScaleFactor.Value : double.NaN;
+
+            int Height = (int)(source.DesiredSize.Height * scale);
+            int Width = (int)(source.DesiredSize.Width * scale);
+            var visual = source.TransformToVisual(null).TransformPoint(new Windows.Foundation.Point(0, 0));
+            int X = (int)(visual.X * scale);
+            int Y = (int)(visual.Y * scale) + 29;
+
+            Bitmap screenshot = new Bitmap(Width, Height);
```

Comment:
Most of the same feedback for WPF applies here, except we can use `@$""` in WinUI.